tests/e2e/deployment-test.sh

```diff
@@ -81,7 +81,7 @@ fi
 if [[ $(gcloud pubsub subscriptions list --filter=name:"${PUBSUB_SUBSCRIPTION}" --format="value(name)") ]]; then
     echo "Subscription [${PUBSUB_SUBSCRIPTION}] already exists, skipping"
 else
-    gcloud pubsub subscriptions create "${PUBSUB_SUBSCRIPTION}" --topic="${PUBSUB_TOPIC}" --ack-deadline=120
+    gcloud pubsub subscriptions create "${PUBSUB_SUBSCRIPTION}" --topic="${PUBSUB_TOPIC}" --ack-deadline=120 --message-retention-duration=86400
```

Comment:
it seem that gcloud cli that is used in E2E test pipeline is in older version anddoes not recognise '--message-retention-duration' param, need to find out why...

Reply:
That's correct, our pipeline was installing latest version of `gcloud` but didn't correctly replace built-in version (from 2018) in PATH. Changes introduced in travis.yaml in this PR are fixing this issue.